gradle/libs.versions.toml

```diff
@@ -93,11 +96,11 @@ androidx-paging-compose = { module = "androidx.paging:paging-compose", version.r
 androidx-profileinstaller = { module = "androidx.profileinstaller:profileinstaller", version.ref = "profileInstaller" }
 androidx-room-compiler = { module = "androidx.room:room-compiler", version.ref = "room" }
 androidx-room-ktx = { module = "androidx.room:room-ktx", version.ref = "room" }
-androidx-room-runtime = { module = "androidx.room:room-runtime", version.ref = "room" }
+androidx-room-runtime = "androidx.room:room-runtime:2.5.2"
```

Comment:
Why does this have a specific version instead of the room value above (2.6.1 now)?

Reply:
The `version-catalog-update-plugin` decided to update this and it's unclear why 🤔  However, this dependency is not used by the project so we can remove it altogether.